### PR TITLE
Move to python3

### PR DESCRIPTION
**Pull request by iSignal**
Use the python3 interpreter.

Remove iteritems which is not in python3

### CODE REVIEW

**Comment by mbautin** [comment]
I would probably keep yb-ctl compatible with Python 2 and Python 3 for now.

File: bin/yb-ctl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (mbautin) [inline bin/yb-ctl]: Let's still keep this as `#!/usr/bin/env python` and make sure it works with Python 2 and Python 3.

**Review by mbautin — CHANGES_REQUESTED** [review]

**Review by kmuthukk — APPROVED** [review]
Thx!

**Comment by ramkumarvs** [comment]
@iSignal i tested the code, it doesn't start for me.

**Pull request merged**